File: PHP/ex1009.php
<?php
$name = trim(fgets(STDIN));
$salary = trim(fgets(STDIN));
$bonus = trim(fgets(STDIN));
$salary = $salary + $bonus * 0.15;

echo "TOTAL = R$ " . number_format($salary, 2, '.', '') . "\n";
?>

File: R/ex1009.r
input <- file('stdin', 'r')

name <- readLines(input, n=1)
salary <- as.double(readLines(input, n=1))
bonus <- as.double(readLines(input, n=1))
total <- salary + (bonus * 0.15)
cat("TOTAL = R$ ", formatC(total, digits = 2, format = "f"), "\n", sep = "")
